<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DEELayerShowOptionWin</class>
 <widget class="QWidget" name="DEELayerShowOptionWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Show Parameters</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	outline:0px;
	color:#DCDCDC;
}

*:disabled{
	background:#444444;
	border-color:#484848;
}

QWidget{
	border:1px solid #DCDCDC;
	background:#484848;
	border-radius:0px;
	color:#DCDCDC;
	border-style:none;
}

QAbstractButton{
	border-radius:0px;
	color:#DCDCDC;
	background:none;
	border-style:none;
}

QAbstractButton:hover{
	color:#FFFFFF;
	background-color:#00BB9E;
}

QAbstractButton:checked,QAbstractButton:pressed{
	color:#DCDCDC;
	border-style:solid;
	border-width:0px 0px 0px 2px;
	padding:4px 4px 4px 2px;
	border-color:#00BB9E;
	background-color:#444444;
}

QLineEdit{
	border:1px solid #242424;
	border-radius:3px;
	padding:2px;
	background-color: rgb(90, 90, 90);
	color:#DCDCDC;
	selection-background-color:#484848;
	selection-background-color: rgba(81, 163, 244);
}

QLineEdit:focus{
	border:1px solid #242424;
}

QLineEdit{
	lineedit-password-character:9679;
}

.QGroupBox{
	border:1px solid #242424;
	border-radius:5px;
	margin-top:3ex;
}

.QGroupBox::title{
	subcontrol-origin:margin;
	position:relative;
	left:10px;
}

.QPushButton,.QToolButton{
	border-style:none;
	border:1px solid #242424;
	color:#DCDCDC;
	padding:5px;
	min-height:15px;
	border-radius:5px;
	background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #484848,stop:1 #383838);
}

.QPushButton:hover{
	background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #646464,stop:1 #525252);
}

.QPushButton:pressed{
	background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #484848,stop:1 #383838);
}

QRadioButton::indicator{
	width:15px;
	height:25px;
}

QRadioButton::indicator::unchecked{
	image:url(:/images/qss/psblack/radiobutton_unchecked.png);
}

QRadioButton::indicator::unchecked:disabled{	
	image:url(:/images/qss/psblack/radiobutton_unchecked_disable.png);
}

QRadioButton::indicator::checked{
	image:url(:/images/qss/psblack/radiobutton_checked.png);
}

QRadioButton::indicator::checked:disabled{
	image:url(:/images/qss/psblack/radiobutton_checked_disable.png);
}

QGroupBox::indicator{
	padding:0px -3px 0px 3px;
}

QCheckBox::indicator,QGroupBox::indicator{
	width:13px;
	height:13px;
}

QCheckBox::indicator:unchecked,QGroupBox::indicator:unchecked{
	image:url(:/images/qss/psblack/checkbox_unchecked.png);
}

QCheckBox::indicator:unchecked:disabled,QGroupBox::indicator:unchecked:disabled{
	image:url(:/images/qss/psblack/checkbox_unchecked_disable.png);
}

QCheckBox::indicator:checked,QGroupBox::indicator:checked{
	image:url(:/images/qss/psblack/checkbox_checked.png);
}

QCheckBox::indicator:checked:disabled,QGroupBox::indicator:checked:disabled{
	image:url(:/images/qss/psblack/checkbox_checked_disable.png);
}

QCheckBox::indicator:indeterminate,QGroupBox::indicator:indeterminate{
	image:url(:/images/qss/psblack/checkbox_parcial.png);
}

QCheckBox::indicator:indeterminate:disabled,QGroupBox::indicator:indeterminate:disabled{
	image:url(:/images/qss/psblack/checkbox_parcial_disable.png);
}

QComboBox{
	border:1px solid #8f8f8f;
	background-color: rgb(118, 118, 118);
	border-radius: 5px;
}

QComboBox::down-arrow{
	image:url(:/images/qss/psblack/add_bottom.png);
	width:10px;
	height:10px;
	right:2px;
}

QComboBox::drop-down{
	subcontrol-origin:padding;
	subcontrol-position:top right;
	width:15px;
	border-left-width:0px;
	border-left-style:solid;
	border-top-right-radius:3px;
	border-bottom-right-radius:3px;
	border-left-color:#242424;
}

QComboBox::drop-down:on{
	top:1px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="mGroupBoxShowBands">
     <property name="title">
      <string>Show Bands</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QRadioButton" name="mRadioBtnGrayScale">
          <property name="text">
           <string>1 Band(GrayScale)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="mComBoxGrayBands"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="mRadioBtnRGB">
        <property name="text">
         <string>3 Bands(RGB)</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="mComBoxRGB_R"/>
        </item>
        <item>
         <widget class="QComboBox" name="mComBoxRGB_G"/>
        </item>
        <item>
         <widget class="QComboBox" name="mComBoxRGB_B"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mGroupBoxImageParams">
     <property name="title">
      <string>Image Stretch</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>5</number>
        </property>
        <item>
         <widget class="QComboBox" name="mComBoxStretchMethod"/>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Min:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="mLineEditMinValue">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Max:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="mLineEditMaxValue">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Opacity:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="DSliderPlus" name="mWidgetOpacityValue" native="true"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QRadioButton" name="mRadioBtnGamma">
          <property name="text">
           <string>Gamma(*/10)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="DSliderPlus" name="mWidgetGammaValue" native="true"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QRadioButton" name="mRadioBtnPalette">
          <property name="text">
           <string>Palette</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="mComBoxPalette"/>
        </item>
        <item>
         <widget class="QComboBox" name="mComBoxLevels">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="mBtnReversePalette">
          <property name="text">
           <string>Reverse</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="mLabelCustom">
          <property name="text">
           <string>Custom:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="mLineEditCustomPalette">
          <property name="placeholderText">
           <string>['','',''...]</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mBtnImport">
       <property name="text">
        <string>Import</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mBtnOk">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mBtnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DSliderPlus</class>
   <extends>QWidget</extends>
   <header>dsliderplus.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images/images.qrc"/>
 </resources>
 <connections/>
</ui>
